advanced/fix_files.sh

```diff
@@ -0,0 +1,15 @@
+# Touch files to ensure they exist (create if non-existing, preserve if existing)
+touch /var/log/pihole-FTL.log /var/log/pihole.log
+touch /run/pihole-FTL.pid /run/pihole-FTL.port
+touch /etc/pihole/dhcp.leases
+mkdir -p /var/run/pihole
+mkdir -p /var/log/pihole
+chown "$USER": /var/run/pihole /var/log/pihole
+# Remove possible leftovers from previous pihole-FTL processes
+rm -f /dev/shm/FTL-* 2> /dev/null
```

Comment:
I don't think we need to redirect output to `/dev/null` as `rm -f` doesn't output anything if the files don't exist.